src/components/Item.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import db from '../lib/firebase';
+
+const Item = ({ name, id, lastPurchasedDate }) => {
+  const checkHandler = () => {
+    db.collection('items').doc(id).update({ lastPurchasedDate: new Date() });
+  };
+
+  const checkDate = (lastPurchasedDate) => {
+    if (lastPurchasedDate === null) {
+      return false;
+    } else {
+      const day = 60 * 60 * 24;
+      const now = Date.now() / 1000;
+      return now - lastPurchasedDate.seconds < day;
+    }
+  };
+  return (
+    <div className="check-item">
+      <input
+        type="checkbox"
+        onChange={checkHandler}
```

Comment:
Even though it is currently checked, if I click on it again, it will update the `lastPurchaseDate`. Consider adding a check inside of `checkHandler` to see if the item is already checked. If it is, then we don't need to update the `lastPurchaseDate`.